src/app/players/base-player/base-player.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, EventEmitter, Input, OnDestroy, Output } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { ConfigService } from 'src/app/services/config/config.service';
+import { PlayerService } from 'src/app/services/player/player.service';
+
+@Component({
+  selector: 'app-base-player',
+  templateUrl: './base-player.component.html',
+  styleUrls: ['./base-player.component.scss']
+})
+export class BasePlayerComponent implements OnDestroy {
+
+  playerConfig: any;
+  isLoading = true;
+  playerSettingconfig: any;
+  private subscription: Subscription;
+  private identifierChangeSubscription: Subscription;
+  @Input() showPlayerOnly = false;
+  @Output() share = new EventEmitter();
+
+  constructor(public configService: ConfigService, public playerService: PlayerService) {
```

Comment:
I dont find any interface methods in the baseComp. These interface methods each player component must implement without fail.